DscResources/MSFT_xMySqlGrant/MSFT_xMySqlGrant.schema.mof

```diff
@@ -2,12 +2,12 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlGrant")] 
 class MSFT_xMySqlGrant : OMI_BaseResource
 { 
-  [Key, Description("Name of MySQL user.")] String UserName;
-  [Key, Description("MySql database name to grant permissions.")] String DatabaseName;
-  [Required, Description("MySql connection credential used for the root."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
-  [Write, Description("MySql user permission type."), ValueMap{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}, Values{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}] String PermissionType;
-  [Write, Description("Ensure given grant to mySql database present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-
+    [Key, Description("Name of MySQL user.")] String UserName;
+    [Key, Description("MySql database name to grant permissions.")] String DatabaseName;
+    [Required, Description("MySql connection credential used for the root."), EmbeddedInstance("MSFT_Credential")] String RootCredential;
+    [Key, Description("MySql user permission type."), ValueMap{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}, Values{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}] String PermissionType;
```

Comment:
@deannastanley would need to chime in about the PermissionType but my guess is we were using the PowerShellOrg schema creator and that's what it sets to if we don't specify a default property and it contains only one value. As I previously suggested we may want to change this to an array it would make sense to modify this back to write.